src/components/Game.js

```diff
@@ -1,17 +1,20 @@
 import React, { Component } from 'react';
 
 import BeforeGame from './BeforeGame';
+import InProgress from './InProgress';
 
 class Game extends Component {
     constructor(props) {
         super(props);
 
         this.onStartGame = this.onStartGame.bind(this);
+        this.onEndGame = this.onEndGame.bind(this);
 
         this.state = {
-            currentGameStatus: 'before',
-            player1: '',
-            player2: ''
+            currentGameStatus: 'inProgress',
+            player1: 'Magda',
```

Comment:
I think, that it is worth to check if everything works properly without those seeds

Reply:
works ok